drivers/amplifiers/ada4250/ada4250.c

```diff
@@ -283,10 +283,10 @@ int32_t ada4250_set_offset(struct ada4250_dev *dev, int32_t offset)
 
 	if (offset < 0) {
 		dev->offset_uv = (-1) * offset_raw * vlsb;
-		return ada4250_write(dev, ADA4250_REG_SNSR_CAL_VAL, (1 << 8 | abs(offset_raw)));
+		return ada4250_write(dev, ADA4250_REG_SNSR_CAL_VAL, offset_raw);
```

Comment:
I would sugest in order to make possible negative offset too
```suggestion
                val = offset > 0 ? 1 << 8 : 0;
                val |= abs(offset_raw);
		return ada4250_write(dev, ADA4250_REG_SNSR_CAL_VAL, val);
```
you can reuse val for the second call.

Reply:
sure, i can do it like that

Reply:
but I need to store also the computed value for the offset in uV:
`dev->offset_uv = (-1) * offset_raw * vlsb;`
or
dev->offset_uv = * offset_raw * vlsb;

and there is no `second call`
it's either offset < 0 or offset >= 0